pysat/tests/test_avg.py

```diff
@@ -255,3 +255,50 @@ def test_2D_avg(self):
         for i, y in enumerate(dummy_y[:-1]):
             check.append(np.all(dummy_val[i, :] == y.astype(int)))
             check.append(np.all(dummy_dev[i, :] == 0))
+
```

Comment:
This is a great start.  There should also be tests of cases where the uneven bins fail.

Reply:
This predates this pull request, but shouldn't line 244 be 'longitude' and not 'mlt'?